portfolio/src/main/webapp/script.js

```diff
@@ -144,7 +145,7 @@ function addComment(name, content, blobKey) {
   if (blobKey !== undefined) {
```

Comment:
Comments without images are showing up with the broken-image thumbnail. I don't remember that happening before, but maybe I missed it. If it's unrelated to auth, feel free to fix in a different PR.

Reply:
I think this is a bug from before. There was suppose to be a line that fixed this behavior earlier but it doesn't seem to be working properly. Will address this issue.

Reply:
Nope I was incorrect on this. Looks like in changing over the implementation I didn't handle the null case that occurred, and needed to check so I wouldn't have a null pointer exception. This has been resolved.

Reply:
It appears this issue is more complicated than I thought. The simple fix I proposed resolved the issue on my local server, but appengine exhibits totally different behavior. I am looking into this.

Reply:
I still see the same behavior, is the fix deployed?

Reply:
Sorry, our comments were in a race condition (: